pkg/webreg/webreg.go

```diff
@@ -613,12 +613,23 @@ Example:
 <p>
 All images built for the release targeted by the job (ex:
 <code>release-4.2</code>) are available under <code>stable:tag-name</code>
-(example: <code>stable:installer</code> or <code>stable:cli</code>). There
-are 2 ways of using these images: creating a registry reference that can be
+(example: <code>stable:installer</code> or <code>stable:cli</code>). If the
+repo is not yet promoting a container image containing the tests you wish to
+run, ensure that promotion is correctly configured. If the container image
+containing the tests should simply contain the repository source or test binaries,
+additional images can be listed in the promotion stanza. Otherwise, the image
+should be declared like any other image in the configuration. This is an example
+of all three approaches:
+</p>
+
+{{ yamlSyntax (index . "imagePromotionConfig") }}
+
+<p>
+There are two ways of using these images: creating a registry reference that can be
 used in your <code>ci-operator</code> config or adding a literal test step to
-the test in your <code>ci-operator</code> config. Images built by ci-operator
+the test in your <code>ci-operator</code> config. Images built by <code>ci-operator</code>
```

Comment:
We can also explain why one would choose the reference (reuse by other tests, attaching a specific command).

Reply:
This is mentioned in the section showing the literal step: https://github.com/openshift/ci-tools/pull/457/files#diff-95aa6a9a4f1daf516567fb2eb073cb3fR657-R661